plugins/client/graphql-kotlin-client-generator/src/main/kotlin/com/expediagroup/graphql/plugin/client/generateClient.kt

```diff
@@ -32,7 +31,7 @@ fun generateClient(
     allowDeprecated: Boolean = false,
     customScalarsMap: List<GraphQLScalar> = emptyList(),
     serializer: GraphQLSerializer = GraphQLSerializer.JACKSON,
-    schema: File,
+    schemaPath: String, // this is the file location, not the contents
```

Comment:
lets remove the comment as it is unnecessary

```suggestion
    schemaPath: String,
```